Composer/packages/webchat-client/src/utils/ConversationService.ts

```diff
@@ -0,0 +1,316 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { v4 as uuidv4 } from 'uuid';
+import axios, { AxiosInstance } from 'axios';
+import { createDirectLine } from 'botframework-webchat';
+
+export const headers = {
```

Comment:
why is this exported?

Reply:
please check the exported stuff down below to make sure they really need to be exported.